internal/section/repository/mariadb/repository_test.go

```diff
@@ -0,0 +1,413 @@
+package repository_test
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"regexp"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/stretchr/testify/assert"
+	"github.com/vinigracindo/mercado-fresco-stranger-strings/internal/section/domain"
+	repository "github.com/vinigracindo/mercado-fresco-stranger-strings/internal/section/repository/mariadb"
+)
+
+func TestSectionRepository_GetAll(t *testing.T) {
+
+	mockSections := []domain.SectionModel{
+		{
+			Id:                 1,
+			SectionNumber:      1,
+			CurrentTemperature: 1,
+			MinimumTemperature: 1,
+			CurrentCapacity:    1,
+			MinimumCapacity:    1,
+			MaximumCapacity:    1,
+			WarehouseId:        1,
+			ProductTypeId:      1,
+		},
+		{
+			Id:                 2,
+			SectionNumber:      2,
+			CurrentTemperature: 2,
+			MinimumTemperature: 2,
+			CurrentCapacity:    2,
+			MinimumCapacity:    2,
+			MaximumCapacity:    2,
+			WarehouseId:        2,
+			ProductTypeId:      2,
+		},
+	}
+
+	rows := sqlmock.NewRows([]string{
+		"id",
+		"sectionNumber",
+		"currentTemperature",
+		"minimumTemperature",
+		"currentCapacity",
+		"minimumCapacity",
+		"maximumCapacity",
+		"warehouseId",
+		"productTypeId",
+	}).AddRow(
+		mockSections[0].Id,
+		mockSections[0].SectionNumber,
+		mockSections[0].CurrentTemperature,
+		mockSections[0].MinimumTemperature,
+		mockSections[0].CurrentCapacity,
+		mockSections[0].MinimumCapacity,
+		mockSections[0].MaximumCapacity,
+		mockSections[0].WarehouseId,
+		mockSections[0].ProductTypeId,
+	).AddRow(
+		mockSections[1].Id,
+		mockSections[1].SectionNumber,
+		mockSections[1].CurrentTemperature,
+		mockSections[1].MinimumTemperature,
+		mockSections[1].CurrentCapacity,
+		mockSections[1].MinimumCapacity,
+		mockSections[1].MaximumCapacity,
+		mockSections[1].WarehouseId,
+		mockSections[1].ProductTypeId,
+	)
+
+	t.Run("should return all sections", func(t *testing.T) {
```

Comment:
Sugestão: em todos os t.Run iniciar a frase sobre o que o teste faz com: "get_all_ok: should return all sections" 
Sugestão também para os demais t.Run